packages/contract-wrappers/src/contract_wrappers/coordinator_wrapper.ts

```diff
@@ -0,0 +1,857 @@
+                expirationTimeSeconds: ([] as BigNumber[]).concat(
+                    Array(rawResponse.signatures.length).fill(rawResponse.expirationTimeSeconds),
+                ),
+            };
+        }
+    }
+
+    private async _getServerEndpointOrThrowAsync(feeRecipientAddress: string): Promise<string> {
+        const coordinatorOperatorEndpoint = await this._registryInstance.getCoordinatorEndpoint.callAsync(
+            feeRecipientAddress,
+        );
+        if (coordinatorOperatorEndpoint === '') {
+            throw new Error(
+                `No Coordinator server endpoint found in Coordinator Registry for feeRecipientAddress: ${feeRecipientAddress}. Registry contract address: ${
+                    this.registryAddress
+                }`,
+            );
+        }
+        return coordinatorOperatorEndpoint;
+    }
+
+    private async _generateSignedZeroExTransactionAsync(
+        data: string,
+        senderAddress: string,
+    ): Promise<SignedZeroExTransaction> {
+        const normalizedSenderAddress = senderAddress.toLowerCase();
+
+        const transaction: ZeroExTransaction = {
+            salt: generatePseudoRandomSalt(),
+            signerAddress: normalizedSenderAddress,
+            data,
+            verifyingContractAddress: this.exchangeAddress,
+        };
+        const transactionHash = transactionHashUtils.getTransactionHashHex(transaction);
+        const signature = await signatureUtils.ecSignHashAsync(
+            this._web3Wrapper.getProvider(),
+            transactionHash,
+            transaction.signerAddress,
+        );
+        return {
+            ...transaction,
+            signature,
+        };
+    }
+
+    private async _executeServerRequestAsync(
+        signedTransaction: SignedZeroExTransaction,
+        txOrigin: string,
+        endpoint: string,
+    ): Promise<CoordinatorServerResponse> {
+        const requestPayload = {
+            signedTransaction,
+            txOrigin,
+        };
+        const response = await fetchAsync(`${endpoint}/v1/request_transaction?networkId=${this.networkId}`, {
+            body: JSON.stringify(requestPayload),
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json; charset=utf-8',
+            },
+        });
+
+        const isError = response.status !== HTTP_OK;
+
+        let json;
+        try {
+            json = await response.json();
+        } catch (e) {
+            // ignore
+        }
+
+        const result = {
+            isError,
+            status: response.status,
+            body: isError ? undefined : json,
+            error: isError ? json : undefined,
+            request: requestPayload,
+            coordinatorOperator: endpoint,
+        };
+
+        return result;
+    }
+
+    private async _submitCoordinatorTransactionAsync(
+        transaction: { salt: BigNumber; signerAddress: string; data: string },
```

Comment:
Let's not in-line type declarations. This might by one of the reasons why your CI doc generation test is failing.